components/tidb_query/src/codec/mysql/json/json_modify.rs

```diff
@@ -28,11 +28,11 @@ impl Json {
         mt: ModifyType,
     ) -> Result<()> {
         if path_expr_list.len() != values.len() {
-            return Err(box_err!("Incorrect parameter count"));
+            return Err(box_err!("Incorrect parameter: {:?}", path_expr_list));
```

Comment:
I think this change makes it less clear - the error is that there is an incorrect number of parameters, not that one of the parameters is incorrect. I think a better message would be something of the form "Incorrect number of parameters; expected: 4, found 3".

Reply:
Thanks for your reply. Fixed.